src/Build/BackEnd/Components/BuildRequestEngine/BuildRequestEngine.cs

```diff
@@ -1326,15 +1292,14 @@ private void IssueBuildRequest(BuildRequestBlocker blocker)
         /// <param name="isLastTask"><code>true</code> if this is the last task for this queue, otherwise <code>false</code>.</param>
         /// <returns>True if the task was scheduled, false otherwise.</returns>
         /// <remarks>This method will return false if an attempt is made to schedule an action after the queue has been shut down.</remarks>
-        private bool QueueAction(Action action, bool isLastTask)
```

Comment:
I assume this was because the value was never used? Should delete the xml comment as well please.

Reply:
Correct on unused. Fixed in next push.